Scripts/utils/zone_interval.py

```diff
@@ -183,7 +183,7 @@ def add(self, orig, dest):
         dest : int
             Tour destination zone number
         """
-        self.matrix.at[self.mapping[orig], self.mapping[dest]] += 1
+        self.matrix.at[self.mapping[orig], self.mapping[dest]] += 1.0
```

Comment:
I do not understand why integers cannot be handled as integers?

Reply:
self.matrix can not be integers here because we will add floating point values to it in other methods. Adding 1 instead of 1.0 here would work as well (implicit conversion from int to float), but I think it's more clear to be consistent with the datatypes.

Reply:
The agent model and the aggregate model behave in very different ways here, the agent model is using `add` and aggregate model `aggregate`. One idea would be to separate them better, keeping integers in the agent version.

Reply:
Is there a good reason to duplicate code because of it? As far as I can see, adding 1 to float64 or int64 values works identically up to 2<sup>53</sup> (or 2<sup>24</sup> for float32). Those values are probably more than we need in this case. If the number range could be an issue in some cases we should use float64.